cmd/monaco/account/download.go

```diff
@@ -29,15 +29,17 @@ import (
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account/downloader"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest"
 	manifestloader "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest/loader"
+	manifestwriter "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest/writer"
 	presistance "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/persistence/account/writer"
 	"github.com/google/uuid"
 	"github.com/spf13/afero"
 	"os"
+	"path/filepath"
 )
 
 func downloadAll(fs afero.Fs, opts *downloadOpts) error {
 	if opts.outputFolder == "" {
-		opts.outputFolder = "project/accounts"
+		opts.outputFolder = "download_account"
```

Comment:
why not "download_account_<timestamp> like for "regular" downloads?

Reply:
It's`download_account_${timestamp}` in case `download_account` already exists. Honestly I thought that's the behavior of config downloads too!

Reply:
Changed to match the behaviour of config downloads